src/modules/main/components/__test__/MainExpertsView.spec.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import { MainExpertsView } from '../MainExpertsView';
+import { store } from '../../../../store/store';
+
+describe('MainExpertsView', () => {
+  it('renders images', () => {
+    render(
+      <Provider store={store}>
+        <MainExpertsView />
+      </Provider>,
+    );
+    const images = screen.queryAllByAltText('doctor');
+    expect(images).toHaveLength(11);
```

Comment:
why exactly 11 ? :)

Reply:
expect for the test must be determined/calculated depending on the test input

Reply:
Fixed